components/script/document_loader.rs

```diff
@@ -130,7 +130,7 @@ impl DocumentLoader {
     pub fn finish_load(&mut self, load: &LoadType) {
         debug!("Removing blocking load {:?} ({}).", load, self.blocking_loads.len());
         let idx = self.blocking_loads.iter().position(|unfinished| *unfinished == *load);
-        self.blocking_loads.remove(idx.expect(&format!("unknown completed load {:?}", load)));
+        idx.unwrap_or_else(|| panic!("unknown completed load {:?}", load));
```

Comment:
We're no longer removing the load?

Also, this could be a bit cleaner IMO just using `match`.

Reply:
Sorry, I should have been clearer, the `self.blocking_loads.remove(...)` should stay.

Thanks for taking this on!

Reply:
Is it fine to change to the following?
```
self.blocking_loads.remove(idx.unwrap_or_else(|| panic!("unknown completed load {:?}", load)));
```

Reply:
Yes, that looks good!